packages/twenty-front/src/modules/domain-manager/states/lastAuthenticatedWorkspaceDomainState.ts

```diff
@@ -4,6 +4,7 @@ import { cookieStorageEffect } from '~/utils/recoil-effects';
 export const lastAuthenticatedWorkspaceDomainState = createState<
   | {
       subdomain: string;
+      hostname?: string;
```

Comment:
I'm not a big fan of having variables that can be underfined. Could we make them all required and return (the SERVER_URL domain all the way from the backend)